packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx

```diff
@@ -154,11 +165,49 @@ const DeviceInfo = (props: { session: SessionWithActivitiesResource }) => {
         <Text
           variant='smallRegular'
           colorScheme='neutral'
-          sx={{ textTransform: 'capitalize' }}
         >
-          {formatRelative(props.session.lastActiveAt || new Date(), new Date())}
+          {getRelativeToNowDate(props.session.lastActiveAt, t)}
         </Text>
       </Col>
     </Flex>
   );
 };
+
+type RelativeDateKeys =
+  | 'relativeDates.previous6DaysAt'
+  | 'relativeDates.lastDayAt'
+  | 'relativeDates.sameDayAt'
+  | 'relativeDates.nextDayAt'
+  | 'relativeDates.next6DaysAt';
+
+const getRelativeToNowDate = (
+  date: Date,
+  t: (localizationKey: string | LocalizationKey | undefined) => string,
```

Comment:
Good thinking passing `t` to this one instead of creating another hook. Shall we do the exact opposite though as its more consistent with the rest of the codebase? 
Let's make `getRelativeToNowDate` return a `localizationKey` and do this instead: `
```tsx
t(relativeDateLocalizationKey({date: lastactiveat, relativeTo: Date.now() }))
```


Nitpicky: we can rename `getRelativeToNowDate` accordingly

Reply:
This is done like this, because there is a case where we have a numeric only value (i.e. `12/12/2022`), that does not need to pass through localization. This case is returned by `formatRelative` along with the other 5 ones. Should we extract this case before calling `t` ?